puppet/zulip_ops/templates/sysctl.d/40-conntrack.conf.erb

```diff
@@ -0,0 +1,2 @@
+# Increase conntrack kernel table size
+net.nf_conntrack_max=<%= [262144, @conntrack_max].max %>
```

Comment:
Why are doing this max calculation here?

Reply:
Unnecessary, merely mirroring the one in `puppet/zulip/templates/sysctl.d/40-uwsgi.conf.erb`:
```
# Allow larger listen backlog
net.core.somaxconn=<%= [128, @somaxconn].max %>
```

I'll drop it.